src/options.cpp

```diff
@@ -456,7 +500,81 @@ bool cOpt::operator!=(const std::string sCompare) const
     return !(*this == sCompare);
 }
 
-void initOptions()
+/** Fill TILESETS mapping with values.
+ * Scans all directores in gfx directory for file named "tileset.txt".
```

Comment:
Yeah. Trying to make this description shorter. Guess it's better to fix that message.